db/seeds.rb

```diff
@@ -1,7 +1,27 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+Highlight.destroy_all
```

Comment:
thought we were using factories instead of seeds?

Reply:
I was using the seeds because i use Postman to query the API a lot.  Do you want me to delete the seeds?  it's not necessary for tests or anything, as factories are.  I just use them for convenience for hitting the API.